### PR TITLE
DOC-9232: Edit / Publish Technical Advisory 114393

### PR DESCRIPTION
**Pull request by florence-crl**
Addresses: DOC-9232

Added Technical Advisory 114393.

### CODE REVIEW

**Review by mgartner — APPROVED** [review]
LGTM

File: src/current/advisories/a114393.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+affected_versions: v23.1.0 to v23.1.12, and pre-release versions v23.1.0-beta.1, v23.1.0-beta.2, v23.1.0-beta.3, v23.1.0-rc.1, and v23.1.0-rc.2
+advisory_date: 2023-11-20
+docs_area: releases
+---
+
+Publication date: {{ page.advisory_date | date: "%B %e, %Y" }}
+
+## Description
+
+A critical bug has been identified in the [optimizer](../v23.1/cost-based-optimizer.html) that may lead to incorrect query results, affecting queries involving [REGIONAL BY ROW tables](../v23.1/regional-tables.html#regional-by-row-tables). The bug can cause rows in the local region of a REGIONAL BY ROW table to be omitted from query results when performing a [locality-optimized](../v23.1/cost-based-optimizer.html#locality-optimized-search-in-multi-region-clusters) [lookup join](../v23.1/joins.html#lookup-joins). The bug manifests under the following conditions:
```
Comment (rmloveland) [inline src/current/advisories/a114393.md:15]: (ignore if this is a rush)

all capitalized mentions of REGIONAL BY ROW tables could be in code font

**Review by rmloveland — APPROVED** [review]
LGTM

File: src/current/advisories/a114393.md (hunk 1 above)
Comment (florence-crl) [reply src/current/advisories/a114393.md]: formatted in code font.

**Pull request merged**